app.js

```diff
@@ -1,219 +0,0 @@
-const bodyParser = require("body-parser");
-const cookieParser = require("cookie-parser");
-const express = require("express");
-const favicon = require("serve-favicon");
-const hbs = require("hbs");
-const mongoose = require("mongoose");
-const logger = require("morgan");
-const path = require("path");
-const session = require("express-session");
-const MongoStore = require("connect-mongo");
-const bcrypt = require("bcrypt");
-const passport = require("passport");
-const LocalStrategy = require("passport-local").Strategy;
-const flash = require("connect-flash");
-const GoogleStrategy = require("passport-google-oauth20").Strategy;
-
-require("dotenv").config();
-
-// MONGOOSE SETUP
-
-const app_name = require("./package.json").name;
-const debug = require("debug")(`${app_name}:${path.basename(__filename).split(".")[0]}`);
-
-// MODEL SETUP
-
-const User = require("./models/user.js");
-const Review = require("./models/review.js");
-const Company = require("./models/company.js");
```

Comment:
We shouldn't be importing db models into our main server initialisation code, we should separate our db logic from this